doc/source/user_guide/utilities.rst

```diff
@@ -113,9 +113,51 @@ index of air.
 .. _grating_equation:
 
 Solving the grating equation
-============================
+==============================
 
 
 The function :py:func:`~.utils.axes.solve_grating_equation` follows the
 conventions described in the tutorial from 
 `Horiba Scientific <https://horiba.com/uk/scientific/products/optics-tutorial/wavelength-pixel-position>`_.
+
+
+.. _centroid:
+
+Calculating the centroid of a spectrum (centre of mass)
+==============================
+
+
+The function :py:meth:`~.signals.luminescence_spectrum.LumiSpectrum.centroid` (based on the utility function :py:func:`~.utils.signals.com`) is an alternative to finding the max intensity of a peak.
```

Comment:
Use line breaks after about 80 characters for better readability of the raw text also in the documentation.
 "max" -> "maximum"